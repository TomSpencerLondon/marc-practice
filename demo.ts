import demo from "./file";

demo();
