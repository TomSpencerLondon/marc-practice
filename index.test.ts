import demo from "./demo";

describe("happy path", () => {
  it("is sane", () => {
    expect(true).toBe(true);
  });

  it("is able to import another file", () => {
    expect(demo()).toBe(true);
  })
})
