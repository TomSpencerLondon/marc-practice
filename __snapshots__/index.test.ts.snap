// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`file returns as a record 1`] = `
MarcRecord {
  "_validationOptions": Object {},
  "fields": Array [],
  "leader": "",
}
`;
